synfig-core/src/synfig/filesystemnative.h

```diff
@@ -54,7 +54,7 @@ namespace synfig
 			friend class FileSystemNative;
 			FILE *file_;
 			ReadStream(FileSystem::Handle file_system, FILE *file);
-			virtual size_t internal_read(void *buffer, size_t size);
```

Comment:
`virtual` and `override` are not mutually exclusive.
Should we keep `virtual`?

Reply:
If I understand correctly it's still virtual:
https://en.cppreference.com/w/cpp/language/virtual
```
If some member function vf is declared as virtual in a class Base, 
and some class Derived, which is derived, directly or indirectly, from Base
...
Then this function in the class Derived is also virtual 
(whether or not the keyword virtual is used in its declaration) 
and overrides Base::vf (whether or not the word override is used in its declaration).
```

This code prints `C` (https://godbolt.org/z/Mnf3vPGaa):
```
#include <iostream>

class A {
    virtual void do_f() = 0; // private member
 public:
    void f() { do_f(); } // public interface
};
struct B : public A {
    void do_f() override { std::cout << "B" << std::endl; }; // overrides A::do_f
};
struct C : public B {
    void do_f() override { std::cout << "C" << std::endl; }; // overrides B::do_f
};

 
int main() {
    C c;
    A* a = &c;
    a->f(); // internally calls C::do_f();
}
```
So I think that we can remove this without any consequences.